src/SharedAssemblyInfo.cs

```diff
@@ -4,5 +4,5 @@
 [assembly: AssemblyCompanyAttribute("Reactive Streams")]
 [assembly: AssemblyCopyrightAttribute("CC0 1.0 Universal")]
 [assembly: AssemblyTrademarkAttribute("")]
-[assembly: AssemblyVersionAttribute("0.0.0.0")]
-[assembly: AssemblyFileVersionAttribute("0.0.0.0")]
+[assembly: AssemblyVersionAttribute("1.0.0.0")]
```

Comment:
Unfortunately that's not possible, these attributes only allow numbers.
But they specify only the version of the generated .dll/.exe file, so the package that we will upload to Nuget will have version 1.0.0.RC1 and the included dll's have 1.0.0.0.